modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/RightSidebar/RightSidebarObjectFieldDetails.tsx

```diff
@@ -95,6 +95,7 @@ export function RightSidebarObjectFieldDetails() {
 
 			delete objectField.defaultValue;
 			delete objectField.listTypeDefinitionId;
+			delete objectField.system;
```

Comment:
Hey Carlos, in reality I ended up realizing that this modification was responsible for correcting the [LPS-198613](https://liferay.atlassian.net/browse/LPS-198613)

Furthermore, I recommend proceeding with removing the code snippet:

```
delete objectField.defaultValue;
delete objectField.listTypeDefinitionId;

```
Analyzing the [DTO](https://github.com/liferay/liferay-portal/blob/master/modules/apps/object/object-admin-rest-impl/rest-openapi.yaml#L159-L227) of the Object Field, these properties are part of the structure expected by the Object Field API. Considering we are making a put, all properties must be sent.

A caveat applies to the defaultValue property, which has a deprecated status in the DTO. But even so, it would not need to be removed before the request, it will just not be used.

Reply:
Another modification that could be made is changing the method to be used in the request, changing it from `PUT` to `PATCH` as in the [example](https://github.com/liferay/liferay-portal/commit/9e30982512bd4248773f716f2b63636989f9fd85).

Note that before the request in the previous example they were deleting the properties:

```
delete objectDefinition.objectRelationships;
delete objectDefinition.objectActions;
delete objectDefinition.objectLayouts;
delete objectDefinition.objectViews;
```

What would happen in data loss with the `PUT` method

Reply:
If you find any justification at the code level that demonstrates the real need to delete these properties, let me know! 😄

Reply:
Hey @igor-franca, thanks for your review. As discussed in person, I am going to close this for now and open a thread in Slack for further analyzes.